packages/create-sitecore-jss/src/common/processes/install.ts

```diff
@@ -30,6 +30,7 @@ export const installPackages = (projectFolder: string, silent?: boolean) => {
       {
         cwd: projectFolder,
         encoding: 'utf8',
+        stdio: 'pipe',
```

Comment:
Isn't ['pipe' the default](https://nodejs.org/api/child_process.html#optionsstdio)? We use ['inherit' in CLI](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-cli/src/spawn.ts#L14), should we use the same here?

Reply:
And 'inherit' didn't work for me

Reply:
Maybe it's working differently because we are using `prompt`...

Reply:
Yeah it's strange because we are using this within other PowerShell-driven scripts (CI) and it seems to work fine.

Reply:
But I used npm 6 for dotnet template, because Node 14 is required. On CI we have npm 8? May be it's connected somehow...

Reply:
That could definitely be ... Have you confirmed this change has no impact on npm 8? I'm fine with this change here as we need to support both npm 6 & 8.

Reply:
But it's related to Node.js, I didn't check using different npm versions, but checked with Node.js

Reply:
Right, but node.js uses different npm by default (node.js 16 > npm 6, node.js > npm 8). Assuming you're using nvm to switch node.js versions, the npm version change would come along implicitly.

Reply:
It does appear to be a Node.js 16 thing, as this issue does not occur on 14. BTW, found we are in fact using [inherit by default](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/common/utils/cmd.ts#L30). We should likely update this instead of setting it here (depending on investigation of Node.js 16 issue).

Reply:
Nice catch! It was a trap... I checked using node 16, and it's working as expected.